tests/test_bears/AspectTestBear.py

```diff
@@ -0,0 +1,27 @@
+from coalib.bearlib.aspects import Root
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+class AspectTestBear(LocalBear, aspects={
+    'detect': [
+        Root.Redundancy.UnusedVariable.UnusedGlobalVariable,
+        Root.Redundancy.UnusedVariable.UnusedLocalVariable,
+    ]
+}, languages=['Python']):
```

Comment:
I find this indentation to be distracting.  I think it would be better if `aspects=` appeared under `LocalBear,` ; and then `languages=` also appearing at the same indent level.
This might mean the length of the aspect class name is too long.  if you need to, go back to using `get_aspect` to keep the line length neat.